city_scrapers/spiders/chi_ssa_28.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa28Spider(CityScrapersSpider):
+    name = "chi_ssa_28"
+    agency = "Chicago Special Service Area #28 Six Corners"
+    timezone = "America/Chicago"
+    start_urls = ["https://sixcorners.com/ssa28"]
+    location = {"name": "Portage Arts Lofts", "address": "4041 N. Milwaukee Ave. #302"}
+
+    def parse(self, response):
+        """Since the meeting dates are in an unordered text block, we'll need to parse
+        them outside of the main parsing loop, and use the list index to iterate through
+        xpath for things like links.
+        """
+        dates = response.xpath('//div[@class="col sqs-col-3 span-3"]/div/div/p[2]/text()').getall()
+        dates = [d.strip(', ')[:-2] for d in dates]
+        dates = [d for d in dates if d]
+        for date in dates:
+            item = response.xpath('//div[@class="col sqs-col-3 span-3"]/div/div/p[2]')
+            meeting = Meeting(
+                title="Six Corners Association Meeting",
+                description="",
+                classification=self._parse_classification(date),
+                start=self._parse_start(item, date),
+                end=self._parse_end(item, date),
```

Comment:
We can just include `None` here and let the pipeline fill in the value

Reply:
@pjsier  - which value specifically?

Reply:
The value of `end`, you can get rid of the `_parse_end` function and just include `end=None` for the kwarg